temporal/api/cloud/auditlog/v1/message.proto

```diff
@@ -0,0 +1,107 @@
+syntax = "proto3";
+
+package temporal.api.cloud.auditlog.v1;
+
+option go_package = "go.temporal.io/api/cloud/auditlog/v1;auditlog";
+option java_package = "io.temporal.api.cloud.auditlog.v1";
+option java_multiple_files = true;
+option java_outer_classname = "MessageProto";
+option ruby_package = "Temporalio::Api::Cloud::AuditLog::V1";
+option csharp_namespace = "Temporalio.Api.Cloud.AuditLog.V1";
+
+import "google/protobuf/timestamp.proto";
+
+// LogRecord represents an audit log entry from Temporal, structured for easy parsing and analysis.
+message LogRecord {
+  // Time when the log was emitted from the source
+  google.protobuf.Timestamp emit_time = 1;
+
+  // Level of the log, i.e. info, warn, error etc
+  LogLevel level = 2;
+
+  // Operator email address
+  string user_email = 3;
+
+  // Operator IP address or server name
+  string caller_ip_address = 4;
+
+  // Operation performed
+  string operation = 5;
+
+  // Detailed information about the operation.
+  OperationDetail details = 6;
+
+  // Operation status from API call
+  string status = 7;
+
+  // Admin or System
+  LogCategory category = 8;
+
+  // Version of the log entry
+  int32 version = 9;
+
+  // Unique ID for the log record.
+  string log_id = 10;
+}
+
+// OperationDetails includes potential more detailed operation logs extracted from the different fields in the
+// Temporal API logs or third party logs, e.g. auth0
+message OperationDetail {
+  // The namespace the operation was performed on
+  string namespace = 1;
+
+  // The email addresses of the users on whom the operation was performed
+  repeated string target_users = 2;
+
+  // Names of any roles processed
+  repeated string roles = 3;
```

Comment:
Is this role ids? Whats an example of operation with role ids?
We do not expose roles or role ids in the new apis..

Reply:
These are roles associated with users, and yes they're part of the older apis which are supported by audit logging. Will be referred to accesses associated with identities in the newer apis